src/node/txreconciliation.cpp

```diff
@@ -56,6 +56,13 @@ class TxReconciliationState
      */
     uint64_t m_k0, m_k1;
 
+    /**
+    * Set of transactions to be added to the reconciliation set on the next trickle. These are still unrequestable for
+    * privacy reasons (to prevent transaction probing), transactions became available (moved to m_local_set) once they
+    * would have been announced via fanout.
```

Comment:
c790aa0e7ebfcf0b03b4d0ef5fcc0ddfa4199943

I think "upon trickle" or "see `m_next_inv_send_time`" is much less confusing.